src/services/bigQuery/format.ts

```diff
@@ -161,7 +161,7 @@ const formatTable = (table: RawGcpBigQueryTable): GcpBigQueryTable => {
     externalDataConfigurationCsvOptionsAllowQuotedNewlines: externalDataConfiguration?.csvOptions?.allowQuotedNewlines || false,
     externalDataConfigurationCsvOptionsAllowJaggedRows: externalDataConfiguration?.csvOptions?.allowJaggedRows || false,
     externalDataConfigurationCsvOptionsEncoding: externalDataConfiguration?.csvOptions?.encoding || '',
-    externalDataConfigurationBigtableOptionsColumnFamilies: externalDataConfiguration?.bigtableOptions?.columnFamilies?.map(
+    externalDataConfigOptionsColumnFamilies: externalDataConfiguration?.bigtableOptions?.columnFamilies?.map(
```

Comment:
maybe we should talk with infra to come up with the best replacement names to not confuse users